scripts/config/syncWorkflows.mjs

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
```

Comment:
Let's add some comment text in here explaining what running this script will do in plain english.

```js
/** 
* This script is for the purpose of keeping a component repository in sync with the latest set of workflows defined for Auro components. 
* Running this script will delete all workflows that exist in the repository and replace them with the latest defined workflows for Auro components. **/
```

No need to use that exact language but something like that. 

Let's add a descriptive section like this to the top of all three files
1. syncAllTemplates
1. syncLinters
1. syncWorkflows

Reply:
Resolved.